app/helpers/application_helper/toolbar/firmware_registry_center.rb

```diff
@@ -1,16 +1,56 @@
 class ApplicationHelper::Toolbar::FirmwareRegistryCenter < ApplicationHelper::Toolbar::Basic
   button_group(
-    'firmware_registry_reload',
+    'firmware_registry_reloading',
     [
       button(
         :firmware_registry_reload,
         'fa fa-refresh fa-lg',
         N_('Refresh this page'),
         N_('Refresh'),
-        :url_parms    => "main_div",
-        :send_checked => true,
-        :klass        => ApplicationHelper::Button::ButtonWithoutRbacCheck
+        :data => {
+          'function'      => 'location.reload()',
```

Comment:
function should be a name of the function, not a call 

can you please change it to `window.location.reload` without `()`?